akka-stream/src/main/mima-filters/2.5.22.backwards.excludes

```diff
@@ -15,3 +15,6 @@ ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.MergeHu
 # #25045 adding Java/Scala interop to SourceQueue and SinkQueue
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.SinkQueueAdapter")
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.SourceQueueAdapter")
+
+# 26896 JavaDSL FlowWithContext
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.javadsl.FlowWithContext.this")
```

Comment:
so far we have used the `2.5.x` file for changes in master

Reply:
I expect this needs backport to 2.5. so I put it here directly